operator/pkg/manifest/operations.go

```diff
@@ -40,6 +40,16 @@ type ChartInfo struct {
 	Overrides    map[string]interface{}
 }
 
+type ResourceLists struct {
+	Target    kube.ResourceList
+	Installed kube.ResourceList
+	Namespace kube.ResourceList
+}
+
+func (r ResourceLists) getWaitForResources() kube.ResourceList {
```

Comment:
```suggestion
func (r *ResourceLists) getWaitForResources() kube.ResourceList {
```
Pointer receiver is more efficient here, you don't have to copy list.

Reply:
I disagree on this, since no state changes are made to the receiver, so I value receiver should be ok. Although pointer receiver is more efficient, copying isn't expensive either. Anyways the returned value is also another concatenated copy of target + ns..

Reply:
Let's merge it first, if there is no big issue for memory footprint, we can leave it like this.